modules/data-entry/src/main/frontend/src/questionnaire/MultipleChoice.jsx

```diff
@@ -332,10 +334,13 @@ function MultipleChoice(props) {
                 // In all other cases, we want to clear the ghost value
                 setGhostValue(GHOST_SENTINEL);
               }
-              updateGhost(value, label);
               acceptEnteredOption(value, label);
               onUpdate && onUpdate(value);
             }}
+            initialSelection={selection.filter(option => option[VALUE_POS])}
+            onRemoveOption={(value, label) => {
+              removeOption(value, label);
+            }}
```

Comment:
This may be the cause of the bug with disappearing default options encountered by @sashaandjic . `removeOption` actually removes the entry from the option list (where option list = list of default options + any new options added by the user via the input). See fix suggestions in the comments on `removeOption`.

Also, this can now be simplified:
```suggestion
            onRemoveOption={removeOption}
```